virttest/qemu_vm.py

```diff
@@ -1055,6 +1055,13 @@ def add_memorys(devices, params):
                 backend_options["size_mem"] = "%sM" % params["mem"]
                 if params.get("vm_mem_backend"):
                     backend_options["backend_mem"] = params.get("vm_mem_backend")
+                if params.get("vm_mem_backend") == "memory-backend-file":
+                    if not params.get("vm_mem_backend_path"):
+                        raise virt_vm.VMConfigMissingError(self.name,
+                                                           'vm_mem_backend_path')
+                    else:
+                        backend_options["mem-path_mem"] = params[
+                            "vm_mem_backend_path"]
```

Comment:
I think I still need to keep this part under line #1058 code section to make sure vm_mem_backend_path will be used when vm_mem_backend is "memory-backend-file"

Reply:
Maybe you didn't get my point that we don't add the else blocker if the procedure raises virt_vm.VMConfigMissingError.
Since the procedure will quit if the condition `not params.get("vm_mem_backend_path")` is Ture,  so the `else` is redundant.

Reply:
updated